packages/@aws-cdk/aws-events-targets/lib/firehose-stream.ts

```diff
@@ -0,0 +1,47 @@
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as firehose from '@aws-cdk/aws-kinesisfirehose';
+import { singletonEventRole } from './util';
+
+/**
+ * Customize the Firehose Stream Event Target
+ */
+export interface FirehoseStreamProps {
```

Comment:
since the module for Kinesis Firehose is called `aws-kinesisfirehose` we should align to that convention. Please add the Kinesis prefix